scripts/agency_fix.py

```
@@ -0,0 +1,364 @@
+
+def run_pytest(extra_args: list[str] | None = None) -> tuple[int, str]:
+    cmd = [sys.executable, "-m", "pytest", "--tb=short", "-q", "--no-header", "--timeout=120"]
+    if extra_args:
+        cmd.extend(extra_args)
+    env = {**os.environ}
+    env.setdefault("API_KEYS", "ci-test-key")
+    env.setdefault("OLLAMA_BASE", "http://localhost:11434")
+    env.setdefault("ROUTER_HEALTH_CHECK_ENABLED", "false")
+    env.setdefault("MONGO_URL", "mongodb://localhost:27017")
+    env.setdefault("DB_NAME", "llm_wiki_dashboard_ci")
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)